pkg/argocd/argocd.go

```diff
@@ -425,13 +431,11 @@ func GetImagesFromApplication(app *v1alpha1.Application) image.ContainerImageLis
 	// The Application may wish to update images that don't create a container we can detect.
 	// Check the image list for images with a force-update annotation, and add them if they are not already present.
 	annotations := app.Annotations
-	if updateImage, ok := annotations[common.ImageUpdaterAnnotation]; ok {
-		for _, img := range *parseImageList(updateImage) {
-			img.ImageTag = nil // the tag from the image list will be a version constraint, which isn't a valid tag
-			if forceStr, force := annotations[fmt.Sprintf(common.ForceUpdateOptionAnnotation, img.ImageAlias)]; force && strings.ToLower(forceStr) == "true" {
-				if images.ContainsImage(img, false) == nil {
-					images = append(images, img)
-				}
+	for _, img := range *parseImageList(annotations) {
+		img.ImageTag = nil // the tag from the image list will be a version constraint, which isn't a valid tag
+		if forceStr, force := annotations[fmt.Sprintf(common.ForceUpdateOptionAnnotation, img.ImageAlias)]; force && strings.ToLower(forceStr) == "true" {
```

Comment:
We do not have an `image.HasForceUpdateOptionAnnotation()` method yet, but maybe we should introduce one for this purpose?